plugin/src/py/android_screenshot_tests/common.py

```diff
@@ -33,7 +33,11 @@ def get_adb():
 
 # a version of subprocess.check_output that returns a utf-8 string
 def check_output(args, **kwargs):
-    return subprocess.check_output(args, **kwargs).decode('utf-8')
+    try:
+        return subprocess.check_output(args, **kwargs).decode('utf-8')
+    except subprocess.CalledProcessError, e:
+        print "Error in subprocess:\n", e
```

Comment:
this will fail in python3 (./gradlew :python:py3Tests should catch this hopefully)